### PR TITLE
fix readme for all-in-one

### PR DESCRIPTION
**Pull request by pavolloffay**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,8 +29,6 @@ because these components attempt to connect to Cassandra right away and hard fai
 Your Agent hostname is `jaeger-agent.${NAMESPACE}.svc.cluster.local`
 
 Once everything is ready, `kubectl get service jaeger-query` tells you where to find Jaeger URL.
-If you are using `minikube` to setup your Kubernetes cluster, the command `minikube service jaeger-query --url`
```
Comment (pavolloffay) [inline README.md]: removed, The same is in dev demplate section and production template will not be probably used with minikube.

File: all-in-one/jaeger-all-in-one-template.yml [hunk 2]
```diff
@@ -12,11 +12,6 @@
 # the License.
 #
 
-# This template uses Jaeger with in-memory storage with limited functionality
```
Comment (objectiser) [inline all-in-one/jaeger-all-in-one-template.yml]: Wondering why this warning is being removed?

Comment (pavolloffay) [inline all-in-one/jaeger-all-in-one-template.yml]: We have a readme with this info. I would like to keep templates as clean as possible. If you want I can bring it back.

Comment (objectiser) [inline all-in-one/jaeger-all-in-one-template.yml]: No that is ok.

**Pull request merged**